src/main/java/uk/gov/hmcts/darts/audio/controller/AudioController.java

```diff
@@ -46,7 +44,7 @@ public class AudioController implements AudioApi {
         securityRoles = {JUDGE, REQUESTER, APPROVER, TRANSCRIBER, TRANSLATION_QA, RCJ_APPEALS},
         globalAccessSecurityRoles = {JUDGE})
     public ResponseEntity<List<AudioMetadata>> getAudioMetadata(Integer hearingId) {
-        List<MediaEntity> mediaEntities = audioTransformationService.getMediaMetadata(hearingId);
+        List<MediaEntity> mediaEntities = audioService.getAudioMetadata(hearingId, 1);
```

Comment:
what is 1? maybe define a const instead?

Reply:
It's channel 1, I don't think it's necessary as it's easily visible in IntelliJ as `channel: 1`

https://github.com/hmcts/darts-api/blob/DMP-2044-getAudioMetadata-C1/src/main/java/uk/gov/hmcts/darts/audio/service/AudioService.java#L12